composer.json

```diff
@@ -7,7 +7,7 @@
   "type": "flarum-extension",
   "license": "MIT",
   "require": {
-    "flarum/core": "^0.1.0-beta.8"
+    "flarum/core": "^0.1.0-beta.13"
```

Comment:
Would `>=0.1.0-beta.12 <0.1.0-beta.14` be a better choice here?

Reply:
We don't use that versioning in FriendsOfFlarum. Also, the Model extender is new in beta 13 so beta 12 wouldn't work either.